app/Services/HomeService.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Product;
+use App\Models\Category;
+use App\Models\ProductImage;
+use App\Utilities\GoogleDriveUtility;
+
+
+class HomeService
+{
+    private $googleDriveUtility;
+    /**
+     * Summary of __construct
+     */
+    public function __construct() 
+    {
+        $this->googleDriveUtility = new GoogleDriveUtility();
+    }
+
+    /**
+     * Summary of getHomeProduct
+     * @return mixed
+     */
+    public function getHomeProduct()
+    {
+        //hitung product kelipatan 4
+        $totalProducts = Product::count();
+        $limit = $totalProducts - ($totalProducts%4);
```

Comment:
Limit static aj gk sih lgsng limit 4, ato 8?

Reply:
Update: limit statci to 8 products